### PR TITLE
Sample data seed

### PR DESCRIPTION
**Pull request by itsOrD**
First PR

### CODE REVIEW

File: client/src/App.jsx [hunk 1]
```diff
@@ -1,24 +1,32 @@
...
     super(props);
     this.state = {
       random: '',
+      price: 'loading...',
     };
-//this bindings
   }
 
-//methods
+  componentDidMount() {
```
Comment (ghost) [inline client/src/App.jsx:14]: I would personally build the fetch logic into a function outside of ComponentDidMount and then just call the func inside of CDM.

File: db/index.js [hunk 2]
```diff
@@ -1,12 +1,69 @@
...
+    if (err) console.log(err);
+    console.log('pricing table ready for new data...', results);
+  });
+
+  const dbSeed = () => {
+    for (let i = 0; i < seedData.pricingArr.length; i++) {
+      let p = seedData.pricingArr[i].price;
+      let l = seedData.pricingArr[i].Listing_id;
+      let queryString = `INSERT INTO pricing (price, Listing_id)
+        VALUES ('${p}', '${l}')`;
```
Comment (ghost) [inline db/index.js:54]: remember to escape the inserts with MySql's built in tools. Refer to Josh's movie list practice that we did, if you forgot. I could be totally wrong too. I'll look into it after I finish reviewing.

**Review by ghost — COMMENTED** [review]
Looks bomb dude, you killed it. Those are just my personal suggestions - pick through what you like and what you don't. Keep it up, you're crushing it.

File: client/src/App.jsx (hunk 1 above)
Comment (itsOrD) [reply client/src/App.jsx:14]: Good idea

File: db/index.js (hunk 2 above)
Comment (itsOrD) [reply db/index.js:54]: Great point. I'll take a look back at it, is super important for the 'inputs' (that are the rest of my components) to not be vulnerable to sql injection.

**Pull request merged**